.github/workflows/multipart/migrate.sh

```diff
@@ -106,6 +106,44 @@ if [ $failed_count_site2 -ne 0 ]; then
 	exit 1
 fi
 
+# Add user group test
+mc admin user add site1 site-replication-issue-user site-replication-issue-password
+mc admin group add site1 site-replication-issue-group site-replication-issue-user
+mc admin group enable site1 site-replication-issue-group
+
+max_wait_attempts=30
+wait_interval=5 
+
+attempt=1
+while true; do
+  diff <(mc admin group info site1 site-replication-issue-group) <(mc admin group info site2 site-replication-issue-group)
+
+  if [[ $? -eq 0 ]]; then
+    echo "Outputs are consistent."
+    break
+  fi
+
+  remaining_attempts=$((max_wait_attempts - attempt))
+  if (( attempt >= max_wait_attempts )); then
+    echo "Outputs remain inconsistent after $max_wait_attempts attempts. Exiting with error."
+    exit 1
+  else
+    echo "Outputs are inconsistent. Waiting for $wait_interval seconds (attempt $attempt/$max_wait_attempts)."
+    sleep $wait_interval
+  fi
+
+  ((attempt++))
+done
+
+status=$(mc admin group info site1 site-replication-issue-group | awk -F': +' '/^Status:/ {print $2}')
```

Comment:
```suggestion
status=$(mc admin group info site1 site-replication-issue-group | awk -F': +' '/^Status:/ {print $2}')
```

use `--json` and then use `jq . -r`